### PR TITLE
[az fleet] updated help examples

### PR DESCRIPTION
**Pull request by Ealianis**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
Updated help text examples for `az fleet`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
az fleet

File: src/fleet/azext_fleet/_help.py [hunk 1]
```diff
@@ -97,21 +117,33 @@
...
+    examples:
+        - name: Show the details of a specific fleet member.
+          text: az fleet member show -g MyFleetResourceGroup -f MyFleetName -n NameOfMember
 """
 
 helps['fleet member delete'] = """
     type: command
     short-summary: Deletes a fleet member.
+    examples:
+        - name: Delete a specific fleet member
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:145]:
```suggestion
        - name: Delete a specific fleet member.
```

File: src/fleet/azext_fleet/_help.py [hunk 2]
```diff
@@ -187,26 +219,42 @@
 helps['fleet updaterun show'] = """
     type: command
     short-summary: Shows a fleet update run.
+    examples:
+        - name: Show the details of a specific updaterun
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:223]:
```suggestion
        - name: Show the details of a specific update run.
```

File: src/fleet/azext_fleet/_help.py [hunk 3]
```diff
@@ -21,50 +21,67 @@
...
+          text: az fleet show -g MyFleetResourceGroup -n MyFleetName
 """
 
 helps['fleet list'] = """
     type: command
-    short-summary: Lists fleets in the specified subscription and resource group.
+    short-summary: Lists all fleets within a resource group.
+    examples:
+        - name: List all fleets that exist within a specific subscription and resource group.
+          text: az fleet list -g MyResourceGroup
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:59]:
```suggestion
        - name: List all fleets that exist within a specific subscription.
          text: az fleet list
        - name: List all fleets that exist within a specific subscription and resource group.
          text: az fleet list -g MyResourceGroup
```

File: src/fleet/azext_fleet/_help.py [hunk 4]
```diff
@@ -187,26 +219,42 @@
...
+    examples:
+        - name: Show the details of a specific updaterun
+          text: az fleet updaterun show -g MyFleetResourceGroup -f MyFleetName -n NameofUpdateRun
 """
 
 helps['fleet updaterun list'] = """
     type: command
     short-summary: Lists the update runs of a fleet.
+    examples:
+        - name: Show the details of a specific updaterun
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:231]:
```suggestion
        - name: Show the details of a specific update run.
```

File: src/fleet/azext_fleet/_help.py [hunk 5]
```diff
@@ -187,26 +219,42 @@
...
+          text: az fleet updaterun list -g MyFleetResourceGroup -f MyFleetName
 """
 
 helps['fleet updaterun delete'] = """
     type: command
     short-summary: Deletes a fleet update run.
+    examples:
+        - name: Delete a specific updaterun.
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:239]:
```suggestion
        - name: Delete a specific update run.
```

File: src/fleet/azext_fleet/_help.py [hunk 6]
```diff
@@ -187,26 +219,42 @@
...
+          text: az fleet updaterun delete -g MyFleetResourceGroup -f MyFleetName -n NameofUpdateRun
 """
 
 helps['fleet updaterun start'] = """
     type: command
     short-summary: Starts a fleet update run.
+    examples:
+        - name: Starts an updaterun
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:247]:
```suggestion
        - name: Starts an update run.
```

File: src/fleet/azext_fleet/_help.py [hunk 7]
```diff
@@ -187,26 +219,42 @@
...
+          text: az fleet updaterun start -g MyFleetResourceGroup -f MyFleetName -n NameofUpdateRun
 """
 
 helps['fleet updaterun stop'] = """
     type: command
     short-summary: Stops a fleet update run.
+    examples:
+        - name: Stops an updaterun
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:255]:
```suggestion
        - name: Stops an update run.
```

File: src/fleet/azext_fleet/_help.py [hunk 8]
```diff
@@ -227,21 +275,33 @@
         - name: --stages
           type: string
           short-summary: Path to a json file that defines the update strategy.
+    examples:
+        - name: Create an updatestrategy from a JSON file.
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:279]:
```suggestion
        - name: Create an update strategy from a JSON file.
```

File: src/fleet/azext_fleet/_help.py [hunk 9]
```diff
@@ -227,21 +275,33 @@
...
+          text: az fleet updatestrategy create -g MyFleetResourceGroup -f MyFleetName -n MyUpdateStrategy --stages MyUpdateStrategyFile.json
 """
 
 helps['fleet updatestrategy show'] = """
     type: command
     short-summary: Shows a update strategy.
+    examples:
+        - name: Show the details of a specific updatestrategy.
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:287]:
```suggestion
        - name: Show the details of a specific update strategy.
```

File: src/fleet/azext_fleet/_help.py [hunk 10]
```diff
@@ -227,21 +275,33 @@
...
+          text: az fleet updatestrategy show -g MyFleetResourceGroup -f MyFleetName -n MyUpdateStrategy
 """
 
 helps['fleet updatestrategy list'] = """
     type: command
     short-summary: Lists the fleet's update strategies.
+    examples:
+        - name: List all updatestratgies for a given fleet.
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:295]:
```suggestion
        - name: List all update strategies for a given fleet.
```

File: src/fleet/azext_fleet/_help.py [hunk 11]
```diff
@@ -227,21 +275,33 @@
...
+          text: az fleet updatestrategy list -g MyFleetResourceGroup -f MyFleetName
 """
 
 helps['fleet updatestrategy delete'] = """
     type: command
     short-summary: Deletes a update strategy.
+    examples:
+        - name: Delete a specific updatestrategy.
```
Comment (jim-minter) [inline src/fleet/azext_fleet/_help.py:303]:
```suggestion
        - name: Delete a specific update strategy.
```

**Review by jim-minter — CHANGES_REQUESTED** [review]

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**